### PR TITLE
Bug 1230390 - Removing Pin option when logpressing a link r=mhenretty

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=85759015d0ef628476370202ea730ac7481fa63e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=63eb4997068875a95ea07c1e9e6458150abf4345

File: apps/system/test/unit/browser_context_menu_test.js [hunk 1]
```diff
@@ -334,30 +334,31 @@ suite('system/BrowserContextMenu', function() {
...
+    test('Does not show pinning when longpressing link', function(done) {
       this.sinon.stub(md1.contextMenuView, 'show', function(items) {
-        assert.isTrue(items[2].id === 'add-to-homescreen');
+        assert.isFalse(items[2].id === 'add-to-homescreen');
+        assert.isFalse(items[2].id === 'pin-to-homescreen');
         done();
       });
       var settingObj = {};
-      settingObj[PINNING_PREF] = false;
+      settingObj[PINNING_PREF] = true;
```
Comment (mikehenrty) [inline apps/system/test/unit/browser_context_menu_test.js]: you can probably just remove this i think since the pref is on be default now.

Comment (albertopq) [inline apps/system/test/unit/browser_context_menu_test.js]: But we are using a mock in the unit test, aren't we?

**Pull request merged**